site/e2e/playwright.config.ts

```diff
@@ -22,7 +29,7 @@ export default defineConfig({
       testMatch: /.*\.spec\.ts/,
       dependencies: ["testsSetup"],
       use: { storageState },
-      timeout: 20_000,
+      timeout: 50_000,
```

Comment:
Otherwise:

```
2024-04-12T07:59:03.3989632Z ==> Errors
2024-04-12T07:59:03.3990075Z [31mTest timeout of 20000ms exceeded.[39m
2024-04-12T07:59:03.3991129Z /home/runner/actions-runner/_work/coder/coder/site/e2e/helpers.ts:357:
2024-04-12T07:59:03.3992071Z    at helpers.ts:357
2024-04-12T07:59:03.3992271Z 
2024-04-12T07:59:03.3992405Z   355 |   });
2024-04-12T07:59:03.3992661Z   356 |
2024-04-12T07:59:03.3993276Z > 357 |   await page.getByTestId("agent-status-ready").waitFor({ state: "visible" });
2024-04-12T07:59:03.3993951Z       |                                                ^
2024-04-12T07:59:03.3994334Z   358 |   return cp;
2024-04-12T07:59:03.3994667Z   359 | };
2024-04-12T07:59:03.3994999Z   360 |
2024-04-12T07:59:03.3995440Z Error: locator.waitFor: Test timeout of 20000ms exceeded.
2024-04-12T07:59:03.3995871Z Call log:
2024-04-12T07:59:03.3996400Z   [2m- waiting for getByTestId('agent-status-ready') to be visible[22m
```

[link](https://productionresultssa11.blob.core.windows.net/actions-results/10e9ed8e-f10d-4b53-abbd-08f57231b0ad/workflow-job-run-cc8517f3-b90c-5e2d-a526-7157b1d71fb8/logs/job/job-logs.txt?rsct=text%2Fplain&se=2024-04-12T08%3A21%3A43Z&sig=RUimU%2Fx4OxB%2FNh7s58OZ%2BLGN3%2FUC2Nl4NobkNcGCDDg%3D&sp=r&spr=https&sr=b&st=2024-04-12T08%3A11%3A38Z&sv=2021-12-02)